src/QandA/components/AnswerList.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import AnswerItem from './AnswerItem.jsx';
+import axios from 'axios';
+import _ from 'underscore';
+
+
+
+class AnswerList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      question_id: this.props.question.question_id,
+      count: 2,
+      answers: [],
+      buttonText: 'See More Answers'
+    }
+    this.fetchAnswers = this.fetchAnswers.bind(this);
+    this.toggleAnswers = this.toggleAnswers.bind(this);
+    this.render = this.render.bind(this);
+    this.renderMoreAnswersButton = this.renderMoreAnswersButton.bind(this);
+  }
+
+  fetchAnswers() {
+    axios.get(`/qa/questions/${this.state.question_id}/answers/?count=99`)
+    .then((data) => {
+      this.setState({
+        answers: data.data
+      })
+    })
+    .catch((err) => {
+      console.log('ANSWER FETCH ERROR: ', err);
+    });
+  }
+
+  toggleAnswers() {
+    if (this.state.count === 2) {
+      this.setState({
+        count: 99,
+        buttonText: 'Collapse Answers'
+      });
+      this.render();
+    } else {
+      this.setState({
+        count: 2,
+        buttonText: 'See More Answers'
+      });
+    }
+  }
+
+  renderMoreAnswersButton() {
+    if (this.state.answers.length && this.state.answers.length > 2) {
```

Comment:
seems a bit repetitive here, could you cut the cruft here? we know this will always be an array as it is initialized as [ ] so wouldn't just this.state.answers.length > 2 work here?